recipes/google-cloud-recommendations-ai/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "google-cloud-recommendations-ai" %}
+{% set version = "0.1.0" %}
```

Comment:
I'm starting with this version (which is not the latest) because it is needed by apache-beam with gcp

Reply:
https://github.com/apache/beam/blob/v2.32.0/sdks/python/setup.py#L202

Reply:
Then why not 0.2.0? In that link it says:
> 'google-cloud-recommendations-ai>=0.1.0,<=0.2.0'

Reply:
Ah, fair.  I hadn't notices the `<=0.2.0`.  Constraints are usually `<` something.  Oops!

I'll update right now.